sys/php/www.conf

```diff
@@ -35,24 +33,25 @@ group = nginx
 ;                            (IPv6 and IPv4-mapped) on a specific port;
 ;   '/path/to/unix/socket' - to listen on a unix socket.
 ; Note: This value is mandatory.
-listen = /run/php-fpm/www.sock
+listen = [::]:9000
```

Comment:
I'd avoid IPv6 ..

Reply:
I've addressed it in https://github.com/PUGX/badge-poser/pull/407